code/modules/materials/material_sheets.dm

```diff
@@ -451,3 +451,13 @@
 	. = ..()
 	amount = max_amount
 	update_icon()
+
+/obj/item/stack/material/graphite
+	name = "graphite"
+	icon_state = "sheet-graphite"
+	default_type = MATERIAL_GRAPHITE
+
+/obj/item/stack/material/graphite/full/Initialize()
+	. = ..()
+	amount = max_amount
+	update_icon()
```

Comment:
this doesn't update the sprite to the ones that have more bars in them. do i need to add something else?

Reply:
you forgot to set icon_has_variants = TRUE

Reply:
thank you!